eo-maven-plugin/src/main/java/org/eolang/maven/Dependencies.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2022 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven;
+
+import com.jcabi.log.Logger;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+import javax.json.JsonValue;
+import org.apache.maven.model.Dependency;
+
+/**
+ * Maven dependencies abstraction.
+ *
+ * @since 0.28.11
+ */
+interface Dependencies {
```

Comment:
@volodya-lombrozo Maybe it can implement `Iterable`? As far as I understand it is just a collection of items.

Reply:
Yes, but I will use `Collections if you don't mind. I need the `stream()` method

Reply:
@mximp Fixed.

Reply:
@volodya-lombrozo with `Collection` you will have to implement all of its methods... and do it in every class. Do you really need such a heavy contract in this case?

Reply:
@mximp I got your idea. No, I don't think we have to implement `Iterable` class - it's an overdesign. We just don't need it. Moreover, I need only `path`.

Reply:
@volodya-lombrozo I agree that `Iterable` would be a better interface for `DcsDepgraph`. When you need `.size()`, just use `LengthOf(Iterable)` from Cactoos.